src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -13,23 +13,28 @@ internal sealed class Conversions : ConversionsBase
         private readonly Binder _binder;
 
         public Conversions(Binder binder)
-            : this(binder, currentRecursionDepth: 0)
+            : this(binder, currentRecursionDepth: 0, includeNullability: false)
         {
         }
 
-        private Conversions(Binder binder, int currentRecursionDepth)
-            : base(binder.Compilation.Assembly.CorLibrary, currentRecursionDepth)
+        private Conversions(Binder binder, int currentRecursionDepth, bool includeNullability)
+            : base(binder.Compilation.Assembly.CorLibrary, currentRecursionDepth, includeNullability)
         {
             _binder = binder;
         }
 
-        protected override ConversionsBase CreateInstance(int currentRecursionDepth)
+        protected override ConversionsBase CreateInstance(int currentRecursionDepth, bool includeNullability)
         {
-            return new Conversions(_binder, currentRecursionDepth);
+            return new Conversions(_binder, currentRecursionDepth, includeNullability);
         }
 
         private CSharpCompilation Compilation { get { return _binder.Compilation; } }
 
+        internal Conversions WithNullability()
```

Comment:
>WithNullability() [](start = 29, length = 17)

Should we assert `!IncludeNullability`?
If not here, then maybe in the `NullableWalker` constructor.

Reply:
Added assert to `NullableWalker`.

---
In reply to: [164573402](https://github.com/dotnet/roslyn/pull/23981#discussion_r164573402) [](ancestors = 164573402)